src/screenComponents/radarView.cpp

```diff
@@ -591,39 +597,58 @@ void GuiRadarView::drawObjects(sf::RenderTarget& window_normal, sf::RenderTarget
     switch(fog_style)
     {
     case NoFogOfWar:
-        foreach(SpaceObject, obj, space_object_list)
+        for(P<SpaceObject> obj : space_object_list)
         {
             visible_objects.insert(*obj);
         }
+
         break;
     case FriendlysShortRangeFogOfWar:
         if (!my_spaceship)
+        {
             return;
-        foreach(SpaceObject, obj, space_object_list)
+        }
+
+        for(P<SpaceObject> obj : space_object_list)
         {
+            // If the object can't hide inside a nebula, show it.
             if (!obj->canHideInNebula())
+            {
                 visible_objects.insert(*obj);
+            }
 
-            if ((!P<SpaceShip>(obj) && !P<SpaceStation>(obj)) || !obj->isFriendly(my_spaceship))
+            // If an object is a ship or station that is friendly, or is our ship...
```

Comment:
Same changes here; consolidate the if statements and set the radius for querying and revealing objects to short-range radar radius on PlayerSpaceships.